docs/AboutUs.adoc

```diff
@@ -4,53 +4,53 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
+Meditabs was developed by the https://se-edu.github.io/docs/Team.html[t12-3] team. +
```

Comment:
We might need to ask the tutor if the link `https://se-edu.github.io/docs/Team.html[t12-3] team' needs to be changed as it is not our team link if I am not mistaken.